R/mathemtical_equations.R

```diff
@@ -1,73 +1,128 @@
+    R = paste0(Recover, r"(\gamma {I} - \xi{R}\\)" )
+  } else 
+  {
+    S = paste0(Susceptible, r"( \mu{N} -)", forceOfInfection, r"(+ \xi{R} - \nu{S}\\)")
+    I = paste0(Infection, forceOfInfection, r"(- \gamma {I} - \nu{I}\\)" )
+    R = paste0(Recover, r"(\gamma {I} - \xi{R} - \nu{R}\\)")
+  }
+  equation <- c(S,I,R)
+  return(equation)
+}
+
+SIRD <- function(mu,forceOfInfection)
+{
+  if (!mu)
+  {
+    S <- paste0(Susceptible, "- ", forceOfInfection)
+    I <- paste0(Infection, forceOfInfection, r"(- \gamma {I} - \delta {I}\\)")
+    R <- paste0(Recover, r"(\gamma {I}\\)")
+  }else 
+  {
+    S <- paste0(Susceptible, r"(\mu_B {N} - \mu_D {S} - )", forceOfInfection)
+    I <- paste0(Infection, forceOfInfection, r"(- \gamma {I} - \delta {I} - \mu_D {I}\\)" )
+    R <- paste0(Recover, r"(\gamma {I} - \mu_D {R}\\)")
+  }
+  D <- paste0(Dead, r"(\delta {I})" )
+  equation <- c(S,I,R,D)
+  return(equation)
+}
+
+SEIR <- function(mu,forceOfInfection)
+{
+  if (!mu)
+  {
+    S <- paste0(Susceptible, "- ", forceOfInfection)
+    E <- paste0(Exposure, forceOfInfection, r"(- \gamma {E}\\)")
+    I <- paste0(Infection, r"(\gamma {E} - \sigma {I}\\)" )
+    R <- paste0(Recover, r"(\sigma {I}\\)")
+  } else 
+  {
+    S <- paste0(Susceptible, r"(\mu_B {N} - \mu_D {S} -)", forceOfInfection)
+    E <- paste0(Exposure, forceOfInfection, r"(- \gamma {E} - \mu_D {E}\\)")
+    I <- paste0(Infection, r"(\gamma {E} - \sigma {I} - \mu_D {I}\\)")
+    R <- paste0(Recover, r"(\sigma {I} - \mu_D {R}\\)")
+  }
+  equation <- c(S,E,I,R)
+  return(equation)
+}
+
+SEIRD <- function(mu,forceOfInfection)
+{
+  if (!mu)
+  {
+    S <- paste0(Susceptible, "- ", forceOfInfection)
+    E <- paste0(Exposure, forceOfInfection, r"(- \gamma {E}\\)")
+    I <- paste0(Infection, r"(\gamma {E} - \sigma {I} - \delta {I}\\)" )
+    R <- paste0(Recover, r"(\sigma {I}\\)")
+  } else 
+  {
+    S <- paste0(Susceptible, r"(\mu_B {N} - \mu_D {S} -)", forceOfInfection)
+    E <- paste0(Exposure, forceOfInfection, r"(- \gamma {E} - \mu_D {E}\\)")
+    I <- paste0(Infection, r"(\gamma {E} - \sigma {I} -  \delta {I} - \mu_D {I}\\)")
+    R <- paste0(Recover, r"(\sigma {I} - \mu_D {R}\\)")
+  }
+  D <- paste0(Dead, r"(\delta {I})" )
+  equation <- c(S,E,I,R,D)
+  return(equation)
+}
+
+SEIRS <- function(mu,forceOfInfection)
+{
+  if (!mu)
+  {
+    S <- paste0(Susceptible, "- ", forceOfInfection, r"(+ \xi {R})")
+    E <- paste0(Exposure, forceOfInfection, r"(- \gamma {E}\\)")
+    I <- paste0(Infection, r"(\gamma {E} - \sigma {I}\\)" )
+    R <- paste0(Recover, r"(\sigma {I} - \xi{R}\\)")
+  } else 
+  {
+    S <- paste0(Susceptible, r"(\mu_B {N} - \mu_D {S} -)", forceOfInfection)
+    E <- paste0(Exposure, forceOfInfection, r"(- \gamma {E} - \mu_D {E}\\)")
+    I <- paste0(Infection, r"(\gamma {E} - \sigma {I} -  \delta {I} - \mu_D {I}\\)")
+    R <- paste0(Recover, r"(\sigma {I} - \mu_D {R}\\)")
+  }
+  equation <- c(S,E,I,R)
+  return(equation)
+}
```

Comment:
Reading over these lines really quickly, not super attentively, I don't see any other appearances of $\nu$, so I'm satisfied that these are likely correct.

Please review the equations after fixing the issue with `\nu` := $\nu$

Reply:
I made the change to the code a few minutes ago, and the mathematical equations for SEIRS with vital dynamics is done. I also change `\nu` to `mu_D`  based on this source [https://docs.idmod.org/projects/emod-hiv/en/2.20_a/model-seir.html](url), in which `\nu` is equivalent to `\mu_D`

Reply:
Okay, I understand another source used a different symbol/variable; please use Ashok's.

I had some trouble accessing [the link target](https://docs.idmod.org/projects/emod-hiv/en/2.20_a/model-seir.html) because the hyperlink was constructed strangely. No fault of your own, just be careful when pasting links that they work.

What change to the code are you referring to? I'm not sure when you say here. Please reply to the correct review comment/thread when you're addressing a particular review item.